lib/i18n/backend/active_record.rb

```diff
@@ -49,29 +49,28 @@ def store_translations(locale, data, options = {})
 
         def lookup(locale, key, scope = [], options = {})
           key = normalize_flat_keys(locale, key, scope, options[:separator])
-          result = Translation.locale(locale).lookup(key)
+          results = Translation.locale(locale).lookup(key).order(key: :desc)
```

Comment:
Why do we need ordering here?
It will just make query more complicated but I don't see any benefits of using it